velox/functions/prestosql/JsonFunctions.h

```diff
@@ -169,6 +169,86 @@ struct JsonArrayGetFunction {
   }
 };
 
+// json_extract_scalar(json, json_path) -> varchar
+// Like json_extract(), but returns the result value as a string (as opposed
+// to being encoded as JSON). The value referenced by json_path must be a scalar
+// (boolean, number or string)
+template <typename T>
+struct JsonExtractScalarFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE bool call(
+      out_type<Varchar>& result,
+      const arg_type<Json>& json,
+      const arg_type<Varchar>& jsonPath) {
+    return callImpl(result, json, jsonPath) == simdjson::SUCCESS;
```

Comment:
nit: about the return type from callImpl being bool and comparing that to simdjson::returntype .